packages/exhibitor-migration-script/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+ls -ltR /pkg
+ls -ltR $PKG_PATH
+
+mkdir -p "$PKG_PATH/usr"
+cp -rp "/pkg/extra/$PKG_NAME" "$PKG_PATH/usr"
+
+mkdir -p "$PKG_PATH/bin"
+ln -s "$PKG_PATH/usr/$PKG_NAME/bin/dcos-exhibitor-migrate-perform" "$PKG_PATH/bin/dcos-exhibitor-migrate-perform"
```

Comment:
`/usr` is explicitly never merged / shared between packages (and will never become a shared folder). Just leaving it in the usr folder is good enough.